### PR TITLE
Data layer refactor of controllers/service-manager

### PR DESCRIPTION
**Pull request by jon-shipley**
Data layer refactor only

### CODE REVIEW

File: admin/controllers/service-manager.js [hunk 1]
```diff
@@ -103,17 +102,17 @@ const setUpdateTiming = async (req, res, next) => {
   }
 
   try {
-    await settings.save()
-    let settingsLog = new SettingsLog()
+    await settingDataService.createOrUpdate(settings)
 
+    let settingsLog = {}
```
Comment (GeorgeChatzigiannis) [inline admin/controllers/service-manager.js]: does it matter if settingslong is an empty object?

File: admin/controllers/service-manager.js [hunk 2]
```diff
@@ -81,11 +80,11 @@ const setUpdateTiming = async (req, res, next) => {
   res.locals.pageTitle = 'Check settings'
   let settings
 
-  const settingsRecord = await Settings.findOne().exec()
+  const settingsRecord = await settingDataService.findOne()
   if (settingsRecord) {
     settings = settingsRecord
   } else {
-    settings = new Settings()
+    settings = {}
```
Comment (GeorgeChatzigiannis) [inline admin/controllers/service-manager.js]: same as the settingsLog is this going to matter if it's an empty object vs undefined?

Comment (GeorgeChatzigiannis) [reply admin/controllers/service-manager.js]: I guess it does cause the query will thrown the exception.
Nevermind this comment

Comment (jon-shipley) [reply admin/controllers/service-manager.js]: Well it gets set later on...

File: admin/controllers/service-manager.js (hunk 1 above)
Comment (jon-shipley) [reply admin/controllers/service-manager.js]: Not at this point

File: admin/services/data-access/setting-log.data.service.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+'use strict'
+
```
Comment (GeorgeChatzigiannis) [inline admin/services/data-access/setting-log.data.service.js]: jsdoc missing

**Pull request merged**